app/models/Reservation.java

```diff
@@ -27,25 +32,33 @@
     @JoinColumn(name = "user_id", referencedColumnName = "id")
     private AppUser user;
 
-    @Column(name = "time_of_reservation")
-    private String timeOfReservation;
+    @Transient
+    private int numberOfPeople;
+
+    @Transient
+    private String time;
+
+    private String date;
+
+    private Long reservationsStart;
```

Comment:
Database columns should be in snake_case (e.g. `reservation_start`), so there should probably be column name annotation on this field.